doc/Language/operators.pod6

```diff
@@ -1602,6 +1602,7 @@ details.
 
 X<Intersection operator>.
 
+'&' as in left hand side arguments 'and' right hand side arguments.
```

Comment:
Please add some subject to this sentence, something like "As a mnemonic... " Or "As an alternative form, this operator uses... "

Reply:
As I'm no native English speaker I wasn't able to come up with a better wording. Initially I had appended it to the previous line but decided to keep that separated to make it stand out more is better.

Reply:
Just use one of the two alternatives I propose...